tutorials/tts/Audio_Codec_Training.ipynb

```diff
@@ -0,0 +1,800 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "7X-TwhdTGmlc"
+      },
+      "source": [
+        "# License"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "fCQUeZRPGnoe"
+      },
+      "source": [
+        "> Copyright (c) 2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.\n",
+        ">\n",
+        "> Licensed under the Apache License, Version 2.0 (the \"License\"); you may not use this file except in compliance with the License. You may obtain a copy of the License at\n",
+        ">\n",
+        "> http://www.apache.org/licenses/LICENSE-2.0\n",
+        ">\n",
+        "> Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an \"AS IS\" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "rtBDkKqVGZJ8"
+      },
+      "source": [
+        "# Introduction"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "pZ2QSsXuGbMe"
+      },
+      "source": [
+        "In this tutorial we show how to use NeMo to train and fine-tune **neural audio codecs**.\n",
+        "\n",
+        "Neural audio codecs are deep learning models that compress audio into a low bitrate representation. The compact embedding space created by these models can be useful for various speech tasks, such as TTS and ASR.\n",
+        "\n",
+        "<div>\n",
+        "<img src=\"https://github.com/NVIDIA/NeMo/releases/download/v1.22.0/nemo_audio_codec.png\" width=\"800\", height=\"400\"/>\n",
```

Comment:
I cannot load the image on browser, is it same on your end? But link is accessible.

Reply:
I just searched "https://github.com/NVIDIA/NeMo/releases" in NeMo and it looks like none of the tutorial images linked from github release work in the browser, because github only provides a download link and not a viewing link. So the images will only display properly in Jupyter and Colab.

Reply:
@titu1994 what is the best solution here?

Reply:
From this [discussion](https://github.com/jupyter/notebook/issues/3278) and my testing, I don't think there is a way to make the image render it github without pushing the image into the repoo.

Beyond the image, there are also other parts of the notebook that also do not render properly in github browser. If someone wants to view it accurately in browser, they probably need to use something like [nbviewer](https://nbviewer.org).